dynawo/sources/Models/Modelica/Dynawo/Electrical/Wind/WECC/WT4BCurrentSource.mo

```diff
@@ -0,0 +1,43 @@
+within Dynawo.Electrical.Wind.WECC;
+
+/*
+* Copyright (c) 2023, RTE (http://www.rte-france.com)
+* See AUTHORS.txt
+* All rights reserved.
+* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, you can obtain one at http://mozilla.org/MPL/2.0/.
+* SPDX-License-Identifier: MPL-2.0
+*
+* This file is part of Dynawo, an hybrid C++/Modelica open source suite of simulation tools for power systems.
+*/
+
+model WT4BCurrentSource "WECC Wind Turbine model without plant controller and with a current source as interface with the grid"
+  import Modelica;
+  import Dynawo;
+
+  extends Dynawo.Electrical.Wind.WECC.BaseClasses.BaseWT4B;
+
+  Modelica.Blocks.Interfaces.RealInput PInjRefPu(start = PInj0Pu) "Active power setpoint at injector terminal in pu (generator convention) (base SNom)" annotation(
```

Comment:
Please align the two inputs with the grid (left border + align with the lines).
Also place them on the icon view, thanks !